src/renderer/components/dialogs/DeltaDialog.tsx

```diff
@@ -1,47 +1,52 @@
 import React, { useState, PropsWithChildren } from 'react'
 import { Dialog, Classes, RadioGroup, Radio } from '@blueprintjs/core'
 import classNames from 'classnames'
-import { DialogProps } from './DialogController'
+
+import type { DialogProps } from '../../contexts/DialogContext'
 
 export const DeltaDialogBase = React.memo<
-  React.PropsWithChildren<{
-    isOpen: boolean
-    onClose: () => void
-    canEscapeKeyClose?: boolean
-    showCloseButton?: boolean
-    fixed?: boolean
-    className?: string
-    style?: React.CSSProperties
-    backdropProps?: any
-    canOutsideClickClose?: boolean
-  }>
->(props => {
-  const isFixed = props.fixed
-  return (
-    <>
-      <Dialog
-        isOpen={props.isOpen}
-        onClose={props.onClose}
-        canOutsideClickClose={
-          typeof props.canOutsideClickClose === 'undefined'
-            ? true
-            : props.canOutsideClickClose
-        }
-        isCloseButtonShown={props.showCloseButton}
-        canEscapeKeyClose={true}
-        backdropProps={props.backdropProps}
-        className={classNames(
-          'delta-dialog',
-          isFixed === true ? 'FixedDeltaDialog' : 'DeltaDialog',
-          [props.className]
-        )}
-        style={props.style}
-      >
-        {props.children}
-      </Dialog>
-    </>
-  )
-})
+  React.PropsWithChildren<
+    {
+      canEscapeKeyClose?: boolean
+      showCloseButton?: boolean
+      fixed?: boolean
+      className?: string
+      style?: React.CSSProperties
+      backdropProps?: React.HTMLProps<HTMLDivElement>
+      canOutsideClickClose?: boolean
+    } & DialogProps
+  >
+>(
+  ({
+    onClose,
+    canEscapeKeyClose = true,
+    canOutsideClickClose = true,
+    showCloseButton = true,
+    ...props
+  }) => {
+    const isFixed = props.fixed
+    return (
+      <>
+        <Dialog
+          isOpen={true}
```

Comment:
This is the only place now where we set `isOpen`. Since every other component is either rendered (opened) or not rendered (closed) we didn't really make use of this property anyhow